Gems/Atom/Feature/Common/Tools/ColorGrading/cmdline/CMD_ColorGradinTools.bat

```diff
@@ -0,0 +1,51 @@
+@echo off
+
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM
+REM
```

Comment:
IS this a spelling mistake in the name of this batch file?  "ColorGradinTools"

Reply:
oops